spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/cloud_database.spec

```diff
@@ -18,6 +18,20 @@ describe ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase do
 
     before { allow(ems).to receive(:with_provider_connection).and_yield(connection) }
 
+    context '#create_cloud_database' do
+      it 'creates the cloud database' do
+        allow(connection).to receive_message_chain(:resource, :manager, :collection, :find).and_return({:id => "test123"})
```

Comment:
Doing that :point_up: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/366/files#r847438036 then you don't need to stub this, you can just do a:
```ruby
context '#create_cloud_database' do
  let(:resource_group) { FactoryBot.create(:resource_group, :name => "test123", :ems_ref => "rg-1", :ext_management_system => ems) }
```

Then pass in `:resource_group_name => resource_group.name` below, and expect `:resource_group   => resource_group.ems_ref` in your `create_resource_instance` test